taxi/tests/test_admin.py

```diff
@@ -0,0 +1,104 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from taxi.models import Car, Manufacturer
+
+
+class DriverAdminTests(TestCase):
+    def setUp(self) -> None:
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin", password="admin123", license_number="ADF12345"
+        )
+        self.client.force_login(self.admin_user)
+
+    def test_driver_license_number_in_list_display(self) -> None:
+        url = reverse("admin:taxi_driver_changelist")
+        res = self.client.get(url)
+        self.assertEqual(res.status_code, 200)
+        self.assertContains(res, self.admin_user.license_number)
+
+    def test_driver_license_number_in_fieldsets(self) -> None:
+        url = reverse("admin:taxi_driver_change", args=(self.admin_user.id,))
+        res = self.client.get(url)
+        self.assertEqual(res.status_code, 200)
+        self.assertContains(res, self.admin_user.license_number)
+
+    def test_driver_add_fieldsets_contains_all_fields(self) -> None:
+        url = reverse("admin:taxi_driver_add")
+        res = self.client.get(url)
+        self.assertEqual(res.status_code, 200)
+        self.assertContains(res, 'name="license_number"')
+        self.assertContains(res, 'name="license_number"')
+        self.assertContains(res, 'name="first_name"')

```

Comment:
The assertion `self.assertContains(res, 'name="license_number"')` is repeated twice. Consider removing the redundant assertion to improve clarity and maintainability of the test.